From 185abfc3d6b4e8f804a3f7216cd8b0459593af87 Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Tue, 26 May 2015 13:03:15 +0200
Subject: [PATCH 1/2] udev: fix inotify handling

This partly reverts:

    commit 6d1b1e0bc6bd020218afc5f05286bf372be283d5
    Author: Tom Gundersen <teg@jklm.no>
    Date:   Sun May 24 15:10:04 2015 +0200

        udevd: worker - fully clean up unnecessary fds

The inotify-fd _is_ used in the workers, so don't close it! Have a look at
udev-watch.c, which keeps track of the inotify-fd as a global variable
(ugh!).
---
 src/udev/udevd.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/udev/udevd.c b/src/udev/udevd.c
index b33a262..87e677e 100644
--- a/src/udev/udevd.c
+++ b/src/udev/udevd.c
@@ -337,7 +337,6 @@ static void worker_spawn(Manager *manager, struct event *event) {
 
                 manager->fd_ep = safe_close(manager->fd_ep);
                 manager->fd_signal = safe_close(manager->fd_signal);
-                manager->fd_inotify = safe_close(manager->fd_inotify);
                 manager->worker_watch[READ_END] = safe_close(manager->worker_watch[READ_END]);
 
                 sigfillset(&mask);
-- 
2.4.1

